### PR TITLE
chore: add first e2e tests

### PR DESCRIPTION
**Pull request by thschue**
## This PR
* adds the first e2e tests using kuttl

### CODE REVIEW

File: .github/workflows/e2e-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      - 'feat/add_tests'
+  pull_request:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+      - 'epic/*'
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - "netlify.toml"
```
Comment (pmig) [inline .github/workflows/e2e-tests.yaml:16]:
```suggestion
    branches:
      - 'main'
  pull_request:
    branches:
      - 'main'
```

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
 /.gradle/
 /**/build/
 /.idea/
+
+/e2e/kubeconfig
```
Comment (pmig) [inline .gitignore:5]:
```suggestion
/e2e/kubeconfig
```

**Review by pmig — CHANGES_REQUESTED** [review]
This seems to be a nice and useful addition of an end to end test.

I found some small copy & paste errors in the workflow file.

Please squash these commits to single one as the commit information is .. let's say .. limited :-)

**Comment by thschue** [comment]
fixed the issues ...

**Pull request merged**